fe/fe-core/src/main/java/org/apache/doris/master/ReportHandler.java

```diff
@@ -1533,7 +1534,7 @@ private static boolean addReplica(long tabletId, TabletMeta tabletMeta, TTabletI
                 if (backendTabletInfo.isSetCooldownMetaId()) {
                     // replica has cooldowned data
                     do {
-                        Pair<Long, Long> cooldownConf = tablet.getCooldownConf();
+                        Pair<Long, Long> cooldownConf = ((LocalTablet) tablet).getCooldownConf();
```

Comment:
If we add a method getCooldownConf in Tablet and implement it in both LocalTablet and CloudTablet, is there extra memory footprint overhead for each tablet object?